src/plots/cartesian/layout_attributes.js

```diff
@@ -60,6 +61,11 @@ module.exports = {
         dflt: '-',
         role: 'info',
         editType: 'calc',
+        // we forget when an axis has been autotyped, just writing the auto
+        // value back to the input - so it doesn't make sense to template this.
+        // Note: we do NOT prohibit this in `coerce`, so if someone enters a
+        // type in the template explicitly it will be honored as the default.
+        _noTemplating: true,
```

Comment:
[`polar.angularaxis.type`](https://github.com/plotly/plotly.js/blob/c2457ff7d2a3189d35a190b9e4bb315f49dc080d/src/plots/polar/layout_attributes.js#L139-L156) does not inherit from `type` in  `cartesian/layout_attributes.js`. Should it also have `_noTemplating: true`?

Reply:
good call - `angularaxis.type` gets `_noTemplating` in https://github.com/plotly/plotly.js/pull/2761/commits/b1c6f0ae0c0d47dab78df8255cd373316a3c3a97